### PR TITLE
Remove BIM Ownership Transferring and address some review comments

### PR DESCRIPTION
**Pull request by shuo-wu**
Directly deleting old managers and ask new managers to reuse the
existing files (if possible) are enough. The side effect of this
modification is that the existing backing image won't be shwon
during the system upgrade, which is acceptable.

### CODE REVIEW

**Comment by shuo-wu** [comment]
I Will test this PR tomorrow.

**Comment by PhanLe1010** [comment]
I like this solution! Feel more resilient than the the transfer ownership gRPC calls.

**Comment by shuo-wu** [comment]
> I like this solution! Feel more resilient than the the transfer ownership gRPC calls.
> Should we also update the LEP?

Sure. I will update the LEP after passing the tests.

**Comment by innobead** [comment]
Can we change to PR instead of Draft PR if it's ready to review? Thanks.

**Comment by shuo-wu** [comment]
Will update the image and vendor after the related backing image manager PR gets merged.

The regression test:
https://ci.longhorn.io/job/private/job/longhorn-tests-regression/151/

File: deploy/install/02-components/01-manager.yaml [hunk 1]
```diff
@@ -31,7 +31,7 @@ spec:
         - --share-manager-image
         - longhornio/longhorn-share-manager:v1_20210406
         - --backing-image-manager-image
-        - longhornio/backing-image-manager:v1_20210407
+        - shuowu/backing-image-manager:v1_20210409
```
Comment (PhanLe1010) [inline deploy/install/02-components/01-manager.yaml:34]: Don't forget to change this one back

**Review by PhanLe1010 — APPROVED** [review]
LGTM. Thanks for the awesome job

**Review by innobead — COMMENTED** [review]
LGTM, but as @PhanLe1010 mentioned, needs to update the backing-image-manager image when ready before merging.

**Review by innobead — APPROVED** [review]
LGTM

**Pull request merged**